src/main/java/uk/gov/hmcts/sscs/email/Email.java

```diff
@@ -2,7 +2,7 @@
 
 import java.util.List;
 
-public class Email {
+public abstract class Email {
```

Comment:
Why abstract class?. Don't see any abstract method.

Reply:
Inheritance is anyway a bad design pattern, and this class cannot be used without a sub-class so the class is marked abstract to make that clear.